src/types.ts

```diff
@@ -10,6 +10,13 @@ export enum CommandType {
   WITNESS_TRANSACTION = 'shelley.transaction.witness',
 }
 
+export enum Era {
```

Comment:
I think `CardanoEra` would be more suitable, `era` seems really general. Also I would refactor in all the other places.